test/tests/functional/pbs_job_purge.py

```diff
@@ -58,12 +58,12 @@ def test_job_files_after_execution(self):
             self.server.pbs_conf['PBS_HOME'], 'mom_priv', 'jobs/')
         # Submit a normal and an array job
         j = Job(TEST_USER)
-        j.create_script(body="sleep 5")
+        j.create_script(body="sleep 20")
```

Comment:
Could you increase this to 30 seconds to avoid race conditions. ?

Reply:
Unless you really need the script, you might choose to use set_sleep_time().  It takes care of paths and other things like that for you.